### DISCUSSION TITLE
Umami docker not starting

### DISCUSSION THREAD

**Discussion by lazerleif** [umami-software/umami#1297] [Q&A]
I just upgraded umami docker. But i wont start. This is from the log. I have no idea how to solve this problem.

`today at 13:02:16yarn run v1.22.19
today at 13:02:16$ npm-run-all check-db start-next
today at 13:02:17$ node scripts/check-db.js
today at 13:02:17✓ DATABASE_URL is defined.
today at 13:02:17✓ Database connection successful.
today at 13:02:17✓ Database tables found.
today at 13:02:18✓ Database is up to date.
today at 13:02:19$ next start
today at 13:02:19/bin/sh: next: not found
today at 13:02:19error Command failed with exit code 127.
today at 13:02:19info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
today at 13:02:19ERROR: "start-next" exited with 127.
today at 13:02:19error Command failed with exit code 1.
today at 13:02:19info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
today at 13:02:18Container stopped`

**Comment by mikecao** [comment]
It says the `next` command is not found. Trying reinstalling your dependencies, `yarn install`.

**Reply by lazerleif** [reply]
I restarted the docker and everything worked. I didn't change anything or so..
